internal/api/server.go

```diff
@@ -21,8 +22,28 @@ import (
 const (
 	V0PathPrefix    = "/v0"
 	V0ApiPathPrefix = "/api" + V0PathPrefix
+	defaultVersion  = "dev"
 )
 
+// getVersion returns the server version string, using the same logic as CLI
+func getVersion() string {
```

Comment:
This code (getVersion & normalizeVersion) is being duplicated from https://github.com/mcpjungle/MCPJungle/blob/main/cmd/version.go.
Instead, we should centralize this logic.

You can create a new package called `version` and put `version.go` inside it. It will contain the version info and utility functions.

Then, both the cmd and server can just use the version package's functionality.